caddyhttp/httpserver/replacer.go

```diff
@@ -41,6 +41,9 @@ var requestReplacer = strings.NewReplacer(
 
 var now = time.Now
 
+// ReplacerCtxKey is the context key for a per-request replacer.
+var ReplacerCtxKey = (caddy.CtxKey)("replacer")
```

Comment:
This syntax is interesting - did you mean `caddy.CtxKey("replacer")`?

Reply:
I would suggest here that we should create a new const in middleware.go with the other context keys.

Reply:
@mholt I've come to like the casting syntax above because it makes the case visually distinct from a function call, but I realize it's unusual. Moving it to middleware.go will avoid the issue entirely, since those keys declare the type to the left of the equals 🙂